src/mcts/node.cc

```diff
@@ -246,6 +249,17 @@ void Node::FinalizeScoreUpdate(float v, int multivisit) {
   n_ += multivisit;
   // Decrement virtual loss.
   n_in_flight_ -= multivisit;
+  // Best child is potentially no longer valid.
+  best_child_cached_ = nullptr;
+}
+
+void Node::UpdateBestChild(const Iterator& best_edge, int visits_allowed) {
+  best_child_cached_ = best_edge.node();
+  // Unexpanded edges can still have nodes. Do not cache unexpanded as they play by different rules.
```

Comment:
Longer than 80 characters. :)

Reply:
FIxed - also fixed some terminology issues in this comment.